### ISSUE TITLE
Slabs don't let redstone signal pass through

### ISSUE THREAD

**Issue by SplendidAlakey** [Micalobia/Full-Slabs#44]
**MC** 1.17.1
**Fabric API** 0.40.1
**FullSlabs** 3.1.4

1. Place a single slab vertically anywhere you want
2. Put a button/lever/any other redstone activator on it
3. Try to power something up

Expected result: power goes through

Actual result: power doesn't go through

Having 2 slabs together (even mixed ones) lets the signal pass through.

Will likely be a problem with horizontal slabs too when #15 is complete.

![2021-10-08_15 46 53](https://user-images.githubusercontent.com/86702831/136568927-a3778889-4126-4b3c-9eac-c6a7bf13d4c9.png)

**Comment by Micalobia** [comment]
This is actually a feature in Vanilla, putting a lever on a top slab doesn't power the slabs next to it, and the same with bottom slabs. This actually happens with stairs as well, and every non-opaque block in the game, despite being able to put levers and buttons on them.

TLDR; It's not an opaque block so it can't be powered in general, and it shares this property with stairs which feels even weirder.

**Comment by SplendidAlakey** [comment]
Oh, shame. I didn't think it was vanilla "feature" because I can't put levers on horizontal slabs at all.

**Comment by Micalobia** [comment]
Yeah, I agree with ya, feels a bit weird that it behaves like that. Patching it would be both *way* outside of the scope of this mod, and just generally break a *lot* of redstone, so it's generally just a bad idea even though it'd be neat.
